pydicom/charset.py

```diff
@@ -95,6 +95,45 @@
                      'iso_ir_58')
 
 
+# shift_jis is a superset of jis_x_0201. So we can regard the encoded value
+# as jis_x_0201 if it is single byte character.
+def _encode_to_jis_x_0201(input, errors='strict'):
```

Comment:
are you planning to use that `errors` argument?

Reply:
I'm planning not to support all the features. I think that it is enough that Only 'strict' and 'replace' are support. Because this functions is designed to be used in charset.py. And there are only 'strict' and 'replace' in charset.py.

Reply:
ok, ignore that - I misread the code